### ISSUE TITLE
Review Volunteer Register Form

### ISSUE THREAD

**Issue by lilianabermudes** [CodeYourFuture/CYF-Staff-Tickets#1280]
### Background

The current form is old, we need to do some updates. There was a discussion in the dashboard team I believe. 

@kfklein15 can we review the current form? 

(should we create a ticket somewhere else?)

### Acceptance criteria

- [ ] Confirm with Karen how we move forward 

### Owner

Halden

### Stakeholder

Karen

### Approver

Liliana

**Comment by kfklein15** [comment]
I reviewed it many moons ago with Alec: https://miro.com/app/board/uXjVK-TpBcM=/?share_link_id=159927266304

Let's go through it in the next sprint?

**Comment by lilianabermudes** [comment]
Yes, please. Can you add Halden and me to it? Thank you!

**Comment by kfklein15** [comment]
Do you want to talk about this in this sprint?
I am stuck with ITD changes, 1 account on dashboard and cohort amendment tickets for now, so not sure we could pick this up in the next 2 months.

**Comment by lilianabermudes** [comment]
No, not necessarily this time - it was your suggestion lol. Let's pick it up whenever you can

**Comment by kfklein15** [comment]
We have these tickets on the Dashboard itself.
